languages/wordpress-indieweb.pot

```diff
@@ -268,153 +268,88 @@ msgid ""
 "\t\t\tother plugins to support additional Indieweb features."
 msgstr ""
 
-#: includes/getting-started.php:15
-msgid ""
-"Once you have activated the Indieweb plugin, review the Options for this "
-"plugin, then complete your user profile, adding in connections to other "
-"websites. This allows you to connect your identity on those sites to your "
-"website."
-msgstr ""
-
 #: includes/getting-started.php:16
 msgid ""
-"Once you have this setup, and have activated Webmentions and Semantic "
-"Linkbacks, you can setup Bridgy using its plugin to connect your blog to "
-"responses from sites such as Facebook and Twitter too, allowing for a "
-"seamless experience."
-msgstr ""
-
-#: includes/getting-started.php:18
-msgid "Plugins"
-msgstr ""
-
-#: includes/getting-started.php:20
-msgid ""
-"For more information on these plugins, visit the <a "
-"href=\"https://indieweb.org/wordpress\" target=\"_blank\">WordPress "
-"page</a> on the IndieWeb wiki"
-msgstr ""
-
-#: includes/getting-started.php:24
-msgid "Send and receive responses with your site"
-msgstr ""
-
-#: includes/getting-started.php:25
-msgid ""
-"Send and receive comments, likes, reposts, and other kinds of post "
-"responses using your own site!"
-msgstr ""
-
-#: includes/getting-started.php:27
-msgid ""
-"<strong>Webmention</strong> <em>(Required)</em> - allows you to send and "
-"receive by adding webmention support to WordPress. Mentions show up as "
-"comments on your site."
-msgstr ""
-
-#: includes/getting-started.php:28
-msgid ""
-"<strong>Semantic Linkbacks</strong> <em>(Required)</em> - makes IndieWeb "
-"comments and mentions look better on your site."
+"Complete your user profile, adding in connections to other websites. This "
+"allows you to connect your identity on those sites to your website."
 msgstr ""
 
-#: includes/getting-started.php:29
+#: includes/getting-started.php:17
 msgid ""
-"<strong>Post Kinds</strong> - Allows you to reply/like/RSVP etc to another "
-"site from your own, by adding support for kinds of posts to WordPress."
-msgstr ""
-
-#: includes/getting-started.php:34
-msgid "Syndicate your content to other sites"
+"Under Options on the Indieweb menu in the sidebar, set whether this is a "
+"single author or multi-author site"
 msgstr ""
 
-#: includes/getting-started.php:35
+#: includes/getting-started.php:18
 msgid ""
-"Linking to the syndicated version of a post allows for bridgy to be able to "
-"discover the post as well as for end users to comment on those sites as a "
-"way of replying."
+"Links to your various sites will now appear invisibly on your site, "
```

Comment:
maybe clarify invisibly with what it does. Something like "Links to your various sites are now inserted into the HTML <head> element."

Reply:
Too technical, I think. But I'll change it slightly.